sdl-core/src/main/scala/io/smartdatalake/definitions/Environment.scala

```diff
@@ -615,6 +615,18 @@ object Environment extends SmartDataLakeLogger {
   }
   private[smartdatalake] var _sdlPlugin: Option[Option[SDLPlugin]] = None
 
+
+  // instantiate sdl plugins if configured. The class names must be separated by a comma. The classname is helpful to map against a configuration list.
+  private[smartdatalake] def sdlPlugins: Option[Seq[SDLPluginWithClassName]] = {
```

Comment:
Why not just a list, which is empty if there are no plugins?
```suggestion
  private[smartdatalake] def sdlPlugins: Seq[SDLPluginWithClassName] = {
```

Reply:
This was to keep the same pattern as sdlPlugin :)

Reply:
Please see https://github.com/smart-data-lake/smart-data-lake/pull/969/commits/7b7cf60085a0e0a0037741ff248f1af75aeb048b